code/modules/research/fabricators.dm

```diff
@@ -81,7 +82,7 @@
 
 /obj/machinery/r_n_d/fabricator/emag()
 	sleep()
-	if(!research_flags &ACCESS_EMAG)
```

Comment:
Yes.
ACCESS_EMAG is a research flag for whatever reason, which the subtype for exofab has.